e2e/testenv/infra/cmd/setup.go

```diff
@@ -13,13 +12,14 @@ import (
 	"sync"
 	"time"
 
-	"github.com/spf13/cobra"
-	"helm.sh/helm/v3/pkg/registry"
-
 	"github.com/rancher/fleet/e2e/testenv"
 	"github.com/rancher/fleet/e2e/testenv/kubectl"
+	"github.com/spf13/cobra"
+	"helm.sh/helm/v3/pkg/registry"
 )
 
+var timeoutDuration = 30 * time.Minute // default timeout duration
```

Comment:
I thought so too, but was curious about other opinions. That being said, I think 5 minutes could make a test fail if it could still recover. Can we agree on 10 minutes?